src/zenml/integrations/mlflow/model_deployers/mlflow_model_deployer.py

```diff
@@ -293,6 +301,8 @@ def find_model_server(
             pipeline_name=pipeline_name or "",
             pipeline_run_id=pipeline_run_id or "",
             pipeline_step_name=pipeline_step_name or "",
+            registry_model_name=registry_model_name or "",
```

Comment:
Any reason why the `registry_model_name` and `registry_model_version` can't be optional strings on the deployment config, it seems like they are optional and just get set to a useless value anyway?